### ISSUE TITLE
Unable to get klipperScreen to start

### ISSUE THREAD

**Issue by markcarroll** [jordanruthe/KlipperScreen#458]
Unable to get klipperScreen to start. WaveShare 5" Touchscreen. RPi 3B+ Ran through the troubleshooting steps. All are set as laid out in there. Still no luck. I just see the console screen on the display. Initially installed with KIAUH but also tried removing and reinstalling manually per some of the other issues on here. no luck. 

```txt
$ cat /var/log/Xorg.0.log
[    11.535] 
X.Org X Server 1.20.11
X Protocol Version 11, Revision 0
[    11.535] Build Operating System: linux Raspbian
[    11.535] Current Operating System: Linux vera 5.10.63-v7+ #1488 SMP Thu Nov 18 16:14:44 GMT 2021 armv7l
[    11.535] Kernel command line: coherent_pool=1M 8250.nr_uarts=0 snd_bcm2835.enable_compat_alsa=0 snd_bcm2835.enable_hdmi=1 video=Composite-1:720x480@60i vc_mem.mem_base=0x3ec00000 vc_mem.mem_size=0x40000000  console=ttyS0,115200 console=tty1 root=PARTUUID=4301748d-02 rootfstype=ext4 fsck.repair=yes rootwait
[    11.535] Build Date: 04 January 2022  10:24:42AM
[    11.535] xorg-server 2:1.20.11-1+rpt1+deb11u1 (https://www.debian.org/support) 
[    11.536] Current version of pixman: 0.40.0
[    11.536] 	Before reporting problems, check http://wiki.x.org
	to make sure that you have the latest version.
[    11.536] Markers: (--) probed, (**) from config file, (==) default setting,
	(++) from command line, (!!) notice, (II) informational,
	(WW) warning, (EE) error, (NI) not implemented, (??) unknown.
[    11.536] (==) Log file: "/var/log/Xorg.0.log", Time: Wed Jan 19 22:40:23 2022
[    11.563] (==) Using system config directory "/usr/share/X11/xorg.conf.d"
[    11.578] (==) No Layout section.  Using the first Screen section.
[    11.578] (==) No screen section available. Using defaults.
[    11.578] (**) |-->Screen "Default Screen Section" (0)
[    11.578] (**) |   |-->Monitor "<default monitor>"
[    11.585] (==) No device specified for screen "Default Screen Section".
	Using the first device section listed.
[    11.585] (**) |   |-->Device "Allwinner A10/A13 FBDEV"
[    11.585] (==) No monitor specified for screen "Default Screen Section".
	Using a default monitor configuration.
[    11.586] (**) Option "Debug" "dmabuf_capable"
[    11.586] (==) Automatically adding devices
[    11.586] (==) Automatically enabling devices
[    11.586] (==) Automatically adding GPU devices
[    11.594] (==) Max clients allowed: 256, resource mask: 0x1fffff
[    11.637] (WW) The directory "/usr/share/fonts/X11/cyrillic" does not exist.
[    11.637] 	Entry deleted from font path.
[    11.637] (WW) The directory "/usr/share/fonts/X11/100dpi/" does not exist.
[    11.637] 	Entry deleted from font path.
[    11.637] (WW) The directory "/usr/share/fonts/X11/75dpi/" does not exist.
[    11.637] 	Entry deleted from font path.
[    11.637] (WW) The directory "/usr/share/fonts/X11/Type1" does not exist.
[    11.637] 	Entry deleted from font path.
[    11.637] (WW) The directory "/usr/share/fonts/X11/100dpi" does not exist.
[    11.637] 	Entry deleted from font path.
[    11.637] (WW) The directory "/usr/share/fonts/X11/75dpi" does not exist.
[    11.637] 	Entry deleted from font path.
[    11.637] (==) FontPath set to:
	/usr/share/fonts/X11/misc,
	built-ins
[    11.638] (==) ModulePath set to "/usr/lib/xorg/modules"
[    11.638] (II) The server relies on udev to provide the list of input devices.
	If no devices become available, reconfigure udev or disable AutoAddDevices.
[    11.638] (II) Loader magic: 0x20cf40
[    11.638] (II) Module ABI versions:
[    11.638] 	X.Org ANSI C Emulation: 0.4
[    11.638] 	X.Org Video Driver: 24.1
[    11.638] 	X.Org XInput driver : 24.1
[    11.638] 	X.Org Server Extension : 10.0
[    12.046] (--) using VT number 2

[    12.046] (II) systemd-logind: logind integration requires -keeptty and -keeptty was not provided, disabling logind integration
[    12.051] (II) xfree86: Adding drm device (/dev/dri/card0)
[    12.052] (II) no primary bus or device found
[    12.052] 	falling back to /sys/devices/platform/soc/soc:gpu/drm/card0
[    12.052] (II) LoadModule: "glx"
[    12.065] (II) Loading /usr/lib/xorg/modules/extensions/libglx.so
[    12.229] (II) Module glx: vendor="X.Org Foundation"
[    12.229] 	compiled for 1.20.11, module version = 1.0.0
[    12.229] 	ABI class: X.Org Server Extension, version 10.0
[    12.229] (II) LoadModule: "fbturbo"
[    12.229] (II) Loading /usr/lib/xorg/modules/drivers/fbturbo_drv.so
[    12.245] (II) Module fbturbo: vendor="X.Org Foundation"
[    12.246] 	compiled for 1.20.11, module version = 0.5.1
[    12.246] 	Module class: X.Org Video Driver
[    12.246] 	ABI class: X.Org Video Driver, version 24.1
[    12.246] (II) FBTURBO: driver for framebuffer: fbturbo
[    12.263] (WW) Falling back to old probe method for fbturbo
[    12.263] (II) Loading sub module "fbdevhw"
[    12.263] (II) LoadModule: "fbdevhw"
[    12.263] (II) Loading /usr/lib/xorg/modules/libfbdevhw.so
[    12.283] (II) Module fbdevhw: vendor="X.Org Foundation"
[    12.283] 	compiled for 1.20.11, module version = 0.0.2
[    12.283] 	ABI class: X.Org Video Driver, version 24.1
[    12.283] (II) FBTURBO(0): using /dev/fb0
[    12.283] (WW) VGA arbiter: cannot open kernel arbiter, no multi-card support
[    12.283] (II) FBTURBO(0): Creating default Display subsection in Screen section
	"Default Screen Section" for depth/fbbpp 16/16
[    12.283] (==) FBTURBO(0): Depth 16, (==) framebuffer bpp 16
[    12.283] (==) FBTURBO(0): RGB weight 565
[    12.283] (==) FBTURBO(0): Default visual is TrueColor
[    12.283] (==) FBTURBO(0): Using gamma correction (1.0, 1.0, 1.0)
[    12.283] (II) FBTURBO(0): hardware: vc4drmfb (video memory: 750kB)
[    12.283] (DB) xf86MergeOutputClassOptions unsupported bus type 0
[    12.283] (**) FBTURBO(0): Option "fbdev" "/dev/fb0"
[    12.283] (**) FBTURBO(0): Option "SwapbuffersWait" "true"
[    12.284] (II) FBTURBO(0): processor: Unknown
[    12.284] (II) FBTURBO(0): checking modes against framebuffer device...
[    12.284] (II) FBTURBO(0): checking modes against monitor...
[    12.284] (II) FBTURBO(0): Virtual size is 800x480 (pitch 800)
[    12.284] (**) FBTURBO(0):  Built-in mode "current"
[    12.284] (==) FBTURBO(0): DPI set to (96, 96)
[    12.284] (II) Loading sub module "fb"
[    12.284] (II) LoadModule: "fb"
[    12.284] (II) Loading /usr/lib/xorg/modules/libfb.so
[    12.309] (II) Module fb: vendor="X.Org Foundation"
[    12.309] 	compiled for 1.20.11, module version = 1.0.0
[    12.309] 	ABI class: X.Org ANSI C Emulation, version 0.4
[    12.825] (II) FBTURBO(0): using backing store heuristics
[    12.839] (II) FBTURBO(0): can't load 'g2d_23' kernel module
[    12.839] (II) FBTURBO(0): failed to enable the use of sunxi display controller
[    12.839] (II) FBTURBO(0): No sunxi-g2d hardware detected (check /dev/disp and /dev/g2d)
[    12.840] (II) FBTURBO(0): G2D hardware acceleration can't be enabled
[    12.840] (II) FBTURBO(0): enabled VFP/NEON optimizations
[    12.840] (==) FBTURBO(0): Backing store enabled
[    12.849] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.849] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.849] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.849] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.849] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.849] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.849] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.849] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.849] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.850] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.851] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.851] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.851] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.851] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.851] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.851] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.851] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.851] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.851] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.851] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.851] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.851] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.851] (EE) FBTURBO(0): FBIOPUTCMAP: Invalid argument
[    12.851] (==) FBTURBO(0): DPMS enabled
[    12.851] (II) FBTURBO(0): failed to enable hardware cursor
[    12.851] (II) FBTURBO(0): no 3D acceleration because the driver has been compiled without libUMP
[    12.851] (II) FBTURBO(0): if this is wrong and needs to be fixed, please check ./configure log
[    12.853] (II) Initializing extension Generic Event Extension
[    12.854] (II) Initializing extension SHAPE
[    12.855] (II) Initializing extension MIT-SHM
[    12.856] (II) Initializing extension XInputExtension
[    12.861] (II) Initializing extension XTEST
[    12.862] (II) Initializing extension BIG-REQUESTS
[    12.863] (II) Initializing extension SYNC
[    12.864] (II) Initializing extension XKEYBOARD
[    12.865] (II) Initializing extension XC-MISC
[    12.866] (II) Initializing extension SECURITY
[    12.867] (II) Initializing extension XFIXES
[    12.868] (II) Initializing extension RENDER
[    12.869] (II) Initializing extension RANDR
[    12.871] (II) Initializing extension COMPOSITE
[    12.873] (II) Initializing extension DAMAGE
[    12.874] (II) Initializing extension MIT-SCREEN-SAVER
[    12.876] (II) Initializing extension DOUBLE-BUFFER
[    12.877] (II) Initializing extension RECORD
[    12.878] (II) Initializing extension DPMS
[    12.879] (II) Initializing extension Present
[    12.880] (II) Initializing extension DRI3
[    12.880] (II) Initializing extension X-Resource
[    12.881] (II) Initializing extension XVideo
[    12.882] (II) Initializing extension XVideo-MotionCompensation
[    12.882] (II) Initializing extension SELinux
[    12.882] (II) SELinux: Disabled on system
[    12.882] (II) Initializing extension GLX
[    12.883] (II) AIGLX: Screen 0 is not DRI2 capable
[    14.864] (II) IGLX: Loaded and initialized swrast
[    14.864] (II) GLX: Initialized DRISWRAST GL provider for screen 0
[    14.864] (II) Initializing extension XFree86-VidModeExtension
[    14.865] (II) Initializing extension XFree86-DGA
[    14.866] (II) Initializing extension XFree86-DRI
[    14.871] (II) Initializing extension DRI2
[    15.156] (II) config/udev: Adding input device generic ft5x06 (79) (/dev/input/event1)
[    15.156] (**) generic ft5x06 (79): Applying InputClass "libinput touchscreen catchall"
[    15.156] (II) LoadModule: "libinput"
[    15.160] (II) Loading /usr/lib/xorg/modules/input/libinput_drv.so
[    15.188] (II) Module libinput: vendor="X.Org Foundation"
[    15.188] 	compiled for 1.20.8, module version = 0.30.0
[    15.188] 	Module class: X.Org XInput Driver
[    15.188] 	ABI class: X.Org XInput driver, version 24.1
[    15.188] (II) Using input driver 'libinput' for 'generic ft5x06 (79)'
[    15.188] (**) generic ft5x06 (79): always reports core events
[    15.188] (**) Option "Device" "/dev/input/event1"
[    15.188] (**) Option "_source" "server/udev"
[    15.240] (II) event1  - generic ft5x06 (79): is tagged by udev as: Touchscreen
[    15.243] (II) event1  - generic ft5x06 (79): device is a touch device
[    15.245] (II) event1  - generic ft5x06 (79): device removed
[    15.320] (**) Option "config_info" "udev:/sys/devices/platform/soc/3f205000.i2c/i2c-11/i2c-10/10-0038/input/input8/event1"
[    15.320] (II) XINPUT: Adding extended input device "generic ft5x06 (79)" (type: TOUCHSCREEN, id 6)
[    15.321] (**) Option "AccelerationScheme" "none"
[    15.321] (**) generic ft5x06 (79): (accel) selected scheme none/0
[    15.321] (**) generic ft5x06 (79): (accel) acceleration factor: 2.000
[    15.321] (**) generic ft5x06 (79): (accel) acceleration threshold: 4
[    15.329] (II) event1  - generic ft5x06 (79): is tagged by udev as: Touchscreen
[    15.332] (II) event1  - generic ft5x06 (79): device is a touch device
[    15.336] (II) config/udev: Adding input device generic ft5x06 (79) (/dev/input/mouse0)
[    15.336] (II) No input driver specified, ignoring this device.
[    15.336] (II) This device may have been added with another device file.
[    15.338] (II) config/udev: Adding input device vc4 (/dev/input/event0)
[    15.339] (**) vc4: Applying InputClass "libinput keyboard catchall"
[    15.339] (II) Using input driver 'libinput' for 'vc4'
[    15.339] (**) vc4: always reports core events
[    15.339] (**) Option "Device" "/dev/input/event0"
[    15.339] (**) Option "_source" "server/udev"
[    15.345] (II) event0  - vc4: is tagged by udev as: Keyboard Pointingstick
[    15.347] (II) event0  - vc4: device is a pointer
[    15.347] (II) event0  - vc4: device is a keyboard
[    15.348] (II) event0  - vc4: device removed
[    15.381] (II) libinput: vc4: needs a virtual subdevice
[    15.381] (**) Option "config_info" "udev:/sys/devices/platform/soc/3f902000.hdmi/rc/rc0/input7/event0"
[    15.381] (II) XINPUT: Adding extended input device "vc4" (type: MOUSE, id 7)
[    15.382] (**) Option "AccelerationScheme" "none"
[    15.382] (**) vc4: (accel) selected scheme none/0
[    15.382] (**) vc4: (accel) acceleration factor: 2.000
[    15.382] (**) vc4: (accel) acceleration threshold: 4
[    15.389] (II) event0  - vc4: is tagged by udev as: Keyboard Pointingstick
[    15.390] (II) event0  - vc4: device is a pointer
[    15.390] (II) event0  - vc4: device is a keyboard
[    15.428] (**) vc4: Applying InputClass "libinput keyboard catchall"
[    15.428] (II) Using input driver 'libinput' for 'vc4'
[    15.428] (**) vc4: always reports core events
[    15.428] (**) Option "Device" "/dev/input/event0"
[    15.428] (**) Option "_source" "_driver/libinput"
[    15.429] (II) libinput: vc4: is a virtual subdevice
[    15.429] (**) Option "config_info" "udev:/sys/devices/platform/soc/3f902000.hdmi/rc/rc0/input7/event0"
[    15.429] (II) XINPUT: Adding extended input device "vc4" (type: KEYBOARD, id 8)
[    15.429] (**) Option "xkb_model" "pc105"
[    15.429] (**) Option "xkb_layout" "us"
[    15.431] (II) config/udev: removing GPU device /sys/devices/platform/soc/soc:gpu/drm/card0 /dev/dri/card0
[    15.431] xf86: remove device 0 /sys/devices/platform/soc/soc:gpu/drm/card0
[    15.439] (II) config/udev: Adding input device generic ft5x06 (79) (/dev/input/mouse0)
[    15.439] (II) No input driver specified, ignoring this device.
[    15.439] (II) This device may have been added with another device file.
[    15.440] (II) config/udev: removing device vc4
[    15.440] (II) UnloadModule: "libinput"
[    15.441] (II) config/udev: removing device vc4
[    15.441] (II) event0  - vc4: device removed
[    15.530] (II) UnloadModule: "libinput"
[    15.532] (II) config/udev: Adding input device vc4 (/dev/input/event0)
[    15.532] (**) vc4: Applying InputClass "libinput keyboard catchall"
[    15.532] (II) Using input driver 'libinput' for 'vc4'
[    15.532] (**) vc4: always reports core events
[    15.532] (**) Option "Device" "/dev/input/event0"
[    15.532] (**) Option "_source" "server/udev"
[    15.537] (II) event0  - vc4: is tagged by udev as: Keyboard Pointingstick
[    15.538] (II) event0  - vc4: device is a pointer
[    15.538] (II) event0  - vc4: device is a keyboard
[    15.539] (II) event0  - vc4: device removed
[    15.570] (II) libinput: vc4: needs a virtual subdevice
[    15.570] (**) Option "config_info" "udev:/sys/devices/platform/soc/3f902000.hdmi/rc/rc0/input7/event0"
[    15.570] (II) XINPUT: Adding extended input device "vc4" (type: MOUSE, id 7)
[    15.571] (**) Option "AccelerationScheme" "none"
[    15.571] (**) vc4: (accel) selected scheme none/0
[    15.571] (**) vc4: (accel) acceleration factor: 2.000
[    15.571] (**) vc4: (accel) acceleration threshold: 4
[    15.576] (II) event0  - vc4: is tagged by udev as: Keyboard Pointingstick
[    15.577] (II) event0  - vc4: device is a pointer
[    15.577] (II) event0  - vc4: device is a keyboard
[    15.578] (**) vc4: Applying InputClass "libinput keyboard catchall"
[    15.578] (II) Using input driver 'libinput' for 'vc4'
[    15.578] (**) vc4: always reports core events
[    15.578] (**) Option "Device" "/dev/input/event0"
[    15.578] (**) Option "_source" "_driver/libinput"
[    15.578] (II) libinput: vc4: is a virtual subdevice
[    15.579] (**) Option "config_info" "udev:/sys/devices/platform/soc/3f902000.hdmi/rc/rc0/input7/event0"
[    15.579] (II) XINPUT: Adding extended input device "vc4" (type: KEYBOARD, id 8)
[    15.579] (**) Option "xkb_model" "pc105"
[    15.579] (**) Option "xkb_layout" "us"
[    15.579] (WW) Option "xkb_variant" requires a string value
[    15.579] (WW) Option "xkb_options" requires a string value
[    15.580] (II) config/udev: removing device generic ft5x06 (79)
[    15.580] (II) event1  - generic ft5x06 (79): device removed
[    15.720] (II) UnloadModule: "libinput"
[    15.722] (II) config/udev: Adding input device generic ft5x06 (79) (/dev/input/event1)
[    15.722] (**) generic ft5x06 (79): Applying InputClass "libinput touchscreen catchall"
[    15.722] (II) Using input driver 'libinput' for 'generic ft5x06 (79)'
[    15.722] (**) generic ft5x06 (79): always reports core events
[    15.722] (**) Option "Device" "/dev/input/event1"
[    15.722] (**) Option "_source" "server/udev"
[    15.731] (II) event1  - generic ft5x06 (79): is tagged by udev as: Touchscreen
[    15.735] (II) event1  - generic ft5x06 (79): device is a touch device
[    15.736] (II) event1  - generic ft5x06 (79): device removed
[    15.850] (**) Option "config_info" "udev:/sys/devices/platform/soc/3f205000.i2c/i2c-11/i2c-10/10-0038/input/input8/event1"
[    15.850] (II) XINPUT: Adding extended input device "generic ft5x06 (79)" (type: TOUCHSCREEN, id 6)
[    15.850] (**) Option "AccelerationScheme" "none"
[    15.850] (**) generic ft5x06 (79): (accel) selected scheme none/0
[    15.850] (**) generic ft5x06 (79): (accel) acceleration factor: 2.000
[    15.850] (**) generic ft5x06 (79): (accel) acceleration threshold: 4
[    15.858] (II) event1  - generic ft5x06 (79): is tagged by udev as: Touchscreen
[    15.861] (II) event1  - generic ft5x06 (79): device is a touch device
[    21.671] (II) event0  - vc4: device removed
[    21.710] (II) event1  - generic ft5x06 (79): device removed
[    21.813] (II) UnloadModule: "libinput"
[    21.813] (II) UnloadModule: "libinput"
[    21.814] (II) UnloadModule: "libinput"
[    21.881] (II) Server terminated successfully (0). Closing log file.
```

Not sure what else to try.

**Comment by alfrix** [comment]
If its an original display use this repo to install your screen: https://github.com/waveshare/LCD-show
If it's a clone use this: https://github.com/goodtft/LCD-show
reboot after installing the driver
check if switching fb0 to fb1 makes it work (or fb1 to fb0)
if you still cannot make it work try a different os image (raspbian lite - mainsailos - fluiddpi) and reinstall
as a last resort if you are still in trouble try the desktop version of raspbian and follow the steps in the docs to make it boot to console (https://klipperscreen.readthedocs.io/en/latest/Installation/#setup-raspberry-pi)

**Comment by markcarroll** [comment]
There is no driver for the WaveShare 5 DSI but there was an [instruction](https://www.waveshare.com/wiki/5inch_DSI_LCD) to add a couple of lines to the `/boot/config.txt` which I did.  No luck. I then found a reference to needing to add the line `needs_root_rights=yes` into /etc/X11/Xwrapper.config which I did.  The screen then went blank a few times looking like it was trying to start. However it eventually goes back to the console screen.  Different errors this time:

```txt
Jan 20 16:47:24 vera xinit[3823]: (II) Server terminated successfully (0). Closing log file.
Jan 20 16:47:24 vera systemd[1]: KlipperScreen.service: Succeeded.
░░ Subject: Unit succeeded
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░ 
░░ The unit KlipperScreen.service has successfully entered the 'dead' state.
Jan 20 16:47:24 vera systemd[1]: KlipperScreen.service: Consumed 3.115s CPU time.
░░ Subject: Resources consumed by unit runtime
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░ 
░░ The unit KlipperScreen.service completed and consumed the indicated resources.
Jan 20 16:47:25 vera systemd[1]: KlipperScreen.service: Scheduled restart job, restart counter is at 118.
░░ Subject: Automatic restarting of a unit has been scheduled
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░ 
░░ Automatic restarting of the unit KlipperScreen.service has been scheduled, as the result for
░░ the configured Restart= setting for the unit.
Jan 20 16:47:25 vera systemd[1]: Stopped KlipperScreen.
░░ Subject: A stop job for unit KlipperScreen.service has finished
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░ 
░░ A stop job for unit KlipperScreen.service has finished.
░░ 
░░ The job identifier is 8592 and the job result is done.
Jan 20 16:47:25 vera systemd[1]: KlipperScreen.service: Consumed 3.115s CPU time.
░░ Subject: Resources consumed by unit runtime
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░ 
░░ The unit KlipperScreen.service completed and consumed the indicated resources.
Jan 20 16:47:25 vera systemd[1]: Started KlipperScreen.
░░ Subject: A start job for unit KlipperScreen.service has finished successfully
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░ 
░░ A start job for unit KlipperScreen.service has finished successfully.
░░ 
░░ The job identifier is 8592.
Jan 20 16:47:25 vera xinit[3851]: X.Org X Server 1.20.11
Jan 20 16:47:25 vera xinit[3851]: X Protocol Version 11, Revision 0
Jan 20 16:47:25 vera xinit[3851]: Build Operating System: linux Raspbian
Jan 20 16:47:25 vera xinit[3851]: Current Operating System: Linux vera 5.10.63-v7+ #1488 SMP Thu Nov 18 16:14:44 GMT 20>
Jan 20 16:47:25 vera xinit[3851]: Kernel command line: coherent_pool=1M 8250.nr_uarts=0 snd_bcm2835.enable_compat_alsa=>
Jan 20 16:47:25 vera xinit[3851]: Build Date: 04 January 2022  10:24:42AM
Jan 20 16:47:25 vera xinit[3851]: xorg-server 2:1.20.11-1+rpt1+deb11u1 (https://www.debian.org/support)
Jan 20 16:47:25 vera xinit[3851]: Current version of pixman: 0.40.0
Jan 20 16:47:25 vera xinit[3851]:         Before reporting problems, check http://wiki.x.org
Jan 20 16:47:25 vera xinit[3851]:         to make sure that you have the latest version.
Jan 20 16:47:25 vera xinit[3851]: Markers: (--) probed, (**) from config file, (==) default setting,
Jan 20 16:47:25 vera xinit[3851]:         (++) from command line, (!!) notice, (II) informational,
Jan 20 16:47:25 vera xinit[3851]:         (WW) warning, (EE) error, (NI) not implemented, (??) unknown.
Jan 20 16:47:25 vera xinit[3851]: (==) Log file: "/var/log/Xorg.0.log", Time: Thu Jan 20 16:47:25 2022
Jan 20 16:47:25 vera xinit[3851]: (==) Using system config directory "/usr/share/X11/xorg.conf.d"
Jan 20 16:47:26 vera xinit[3852]: modprobe: FATAL: Module g2d_23 not found in directory /lib/modules/5.10.63-v7+
Jan 20 16:47:28 vera xinit[3864]: RuntimeError: module compiled against API version 0xf but this version of numpy is 0xe
Jan 20 16:47:28 vera xinit[3864]: Traceback (most recent call last):
Jan 20 16:47:28 vera xinit[3864]:   File "/home/pi/KlipperScreen/screen.py", line 40, in <module>
Jan 20 16:47:28 vera xinit[3864]:     import matplotlib.pyplot
Jan 20 16:47:28 vera xinit[3864]:   File "/home/pi/.KlipperScreen-env/lib/python3.9/site-packages/matplotlib/__init__.p>
Jan 20 16:47:28 vera xinit[3864]:     from . import _api, _version, cbook, docstring, rcsetup
Jan 20 16:47:28 vera xinit[3864]:   File "/home/pi/.KlipperScreen-env/lib/python3.9/site-packages/matplotlib/rcsetup.py>
Jan 20 16:47:28 vera xinit[3864]:     from matplotlib.colors import Colormap, is_color_like
Jan 20 16:47:28 vera xinit[3864]:   File "/home/pi/.KlipperScreen-env/lib/python3.9/site-packages/matplotlib/colors.py">
Jan 20 16:47:28 vera xinit[3864]:     from matplotlib import _api, cbook, scale
Jan 20 16:47:28 vera xinit[3864]:   File "/home/pi/.KlipperScreen-env/lib/python3.9/site-packages/matplotlib/scale.py",>
Jan 20 16:47:28 vera xinit[3864]:     from matplotlib.ticker import (
Jan 20 16:47:28 vera xinit[3864]:   File "/home/pi/.KlipperScreen-env/lib/python3.9/site-packages/matplotlib/ticker.py">
Jan 20 16:47:28 vera xinit[3864]:     from matplotlib import transforms as mtransforms
Jan 20 16:47:28 vera xinit[3864]:   File "/home/pi/.KlipperScreen-env/lib/python3.9/site-packages/matplotlib/transforms>
Jan 20 16:47:28 vera xinit[3864]:     from matplotlib._path import (
Jan 20 16:47:28 vera xinit[3864]: ImportError: numpy.core.multiarray failed to import
Jan 20 16:47:29 vera xinit[3850]: /usr/bin/xinit: connection to X server lost
Jan 20 16:47:29 vera xinit[3850]: 
Jan 20 16:47:29 vera xinit[3850]: waiting for X server to shut down
Jan 20 16:47:29 vera xinit[3851]: (II) Server terminated successfully (0). Closing log file.
Jan 20 16:47:30 vera systemd[1]: KlipperScreen.service: Succeeded.
░░ Subject: Unit succeeded
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░ 
░░ The unit KlipperScreen.service has successfully entered the 'dead' state.
Jan 20 16:47:30 vera systemd[1]: KlipperScreen.service: Consumed 3.158s CPU time.
░░ Subject: Resources consumed by unit runtime
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░ 
░░ The unit KlipperScreen.service completed and consumed the indicated resources.
```

I'm not a python expert but I tried `pip3 install -U numpy` but that didn't help.  It appears I have lumpy version 1.22.1.

Any other ideas?

**Comment by alfrix** [comment]
with the log i can tell, that your issue is related with the recent issue with numpy 1.21.5, we had to downgrade because it doesn't work with debian buster, you are running bullseye, try this:

```
cd /home/pi/.KlipperScreen-env
source bin/activate  
pip cache purge
pip uninstall matplotlib
pip uninstall numpy
deactivate 
```
then run KlipperScreen-install

**Comment by alfrix** [comment]
Fixed with d2419335b48cb43d18ef627faa5c1b8dabdcf792 just update or reinstall

**Comment by markcarroll** [comment]
Fantastic - that did it. Thanks for the fix.
